tools/orchestrator/package-lock.json

```diff
@@ -21,7 +21,7 @@
     },
     "node_modules/@types/mkdirp": {
       "version": "2.0.0",
-      "resolved": "https://repox.jfrog.io/artifactory/api/npm/npm/@types/mkdirp/-/mkdirp-2.0.0.tgz",
+      "resolved": "https://registry.npmjs.org/@types/mkdirp/-/mkdirp-2.0.0.tgz",
```

Comment:
Dropped references from our private repox, for external contributors to be able to install deps